GitUI/Editor/FileViewer.cs

```diff
@@ -1756,116 +1699,32 @@ private void CopyPatchToolStripMenuItemClick(object sender, EventArgs e)
 
         private void copyNewVersionToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            CopyNotStartingWith('-');
+            internalFileViewer.CopyNotStartingWith('-');
         }
 
         private void copyOldVersionToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            CopyNotStartingWith('+');
+            internalFileViewer.CopyNotStartingWith('+');
         }
 
         /// <summary>
         /// Go to next change
-        /// For normal diffs, this is the next diff
+        /// For normal diffs, this is the next block of lines with a difference.
         /// For range-diff, it is the next commit summary header.
         /// </summary>
-        /// <param name="sender">sender object.</param>
-        /// <param name="e">event args.</param>
         private void NextChangeButtonClick(object sender, EventArgs e)
         {
             Focus();
 
```

Comment:
```suggestion
```

Reply:
Is the control really always in focus here?